src/status_im/contexts/wallet/bridge/flow_config.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im.contexts.wallet.bridge.flow-config)
+
+(def steps
```

Comment:
is this right? there is surely more steps than just these two.
cc @briansztamfater

Reply:
@J-Son89 I don't think all the steps is needed. We need the steps where we call `wizard-navigate` which is for 2 screens only. The steps for the send flow I think was added by @mmilad75 and I asked him to review it here https://github.com/status-im/status-mobile/pull/19860#discussion_r1587090726

Reply:
Hmm I would disagree, this way you are mixing solutions which is a bit odd. imo it's cleaner to use one approach for consistency.

Further to that it's missing some steps which are dynamic. E.g we need to select which account we are sending 'from' which is the first step. I.e if you go to bridge from the home page by long pressing an asset and pressing bridge.

Reply:
I think we need to add all the steps in the wizard config, if we don't use `wizard-navigate` and instead we use navigation events (like `navigate-to-within-stack`) directly, we would need to modify those calls to use `wizard-navigate` instead.

Reply:
Fixed! 👍